src/Components/ResourceList/Card/ResourceCardMeta.tsx

```diff
@@ -0,0 +1,72 @@
+// Libraries
+import React, {forwardRef} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {FlexBox, FlexBoxProps, FlexBoxRef} from '../../FlexBox'
+
+// Types
+import {Omit, FlexDirection, AlignItems} from '../../../Types'
+
+// Styles
+import './ResourceCardMeta.scss'
+
+export interface ResourceCardMetaProps
+  extends Omit<
+    FlexBoxProps,
+    'stretchToFitWidth' | 'stretchToFitHeight' | 'margin'
+  > {}
+
+export type ResourceCardMetaRef = FlexBoxRef
+
+export const ResourceCardMeta = forwardRef<
+  ResourceCardMetaRef,
+  ResourceCardMetaProps
+>(
+  (
+    {
+      id,
+      style,
+      testID = 'resource-list--meta',
+      children,
+      className,
+      direction = FlexDirection.Row,
+      alignItems = AlignItems.Center,
+      justifyContent,
+    },
+    ref
+  ) => {
+    const resourceCardMetaClass = classnames('cf-resource-meta', {
+      [`${className}`]: className,
+    })
+
+    let wrappedChildren = children
+
+    if (React.Children.count(children) > 0) {
+      const childArray: JSX.Element[] = []
+
+      React.Children.forEach(children, child => {
+        childArray.push(<div className="cf-resource-meta--item">{child}</div>)
+      })
```

Comment:
style suggestion, doesn't need to be taken. but the kids apparently like `map` better than `forEach` these days.

```suggestion
      const childArray: JSX.Element[] = React.Children.map(children, child => {
        return(<div className="cf-resource-meta--item">{child}</div>)
      })
```